ci-operator/step-registry/hypershift/mce/kubevirt/destroy/hypershift-mce-kubevirt-destroy-chain.yaml

```diff
@@ -10,23 +10,35 @@ chain:
           source "${SHARED_DIR}/proxy-conf.sh"
         fi
 
+        MCE_VERSION=$(oc get "$(oc get multiclusterengines -oname)" -ojsonpath="{.status.currentVersion}" | cut -c 1-3)
+        HYPERSHIFT_NAME=hcp
+        if (( $(echo "$MCE_VERSION < 2.4" | bc -l) )); then
+          echo "MCE version is less than 2.4, use hypershift command as default"
+          HYPERSHIFT_NAME=hypershift
+        else
+          echo "MCE version is greater than or equal to 2.4, need to extract HyperShift cli for dump command"
+          oc extract secret/pull-secret -n openshift-config --to=/tmp --confirm
+          HO_IMAGE=$(oc get deployment -n hypershift operator -ojsonpath='{.spec.template.spec.containers[*].image}')
+          oc image extract "${HO_IMAGE}" --path /usr/bin/hypershift:/tmp --registry-config=/tmp/.dockerconfigjson
+        fi
+
         arch=$(arch)
         if [ "$arch" == "x86_64" ]; then
-          downURL=$(oc get ConsoleCLIDownload hypershift-cli-download -o json | jq -r '.spec.links[] | select(.text | test("Linux for x86_64")).href') && curl -k --output /tmp/hypershift.tar.gz ${downURL}
-          cd /tmp && tar -xvf /tmp/hypershift.tar.gz
-          chmod +x /tmp/hypershift
+          downURL=$(oc get ConsoleCLIDownload ${HYPERSHIFT_NAME}-cli-download -o json | jq -r '.spec.links[] | select(.text | test("Linux for x86_64")).href') && curl -k --output /tmp/${HYPERSHIFT_NAME}.tar.gz ${downURL}
+          cd /tmp && tar -xvf /tmp/${HYPERSHIFT_NAME}.tar.gz
+          chmod +x /tmp/${HYPERSHIFT_NAME}
           cd -
         fi
 
         CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
-        
+
         /tmp/hypershift dump cluster --artifact-dir=$ARTIFACT_DIR \
```

Comment:
we can remove this part, AFAIK hcp don't have dump, we can use `hypershift-mce-dump` step in the workflow